yr/tutorials/AlexaHelloWorld.md

```diff
@@ -94,61 +111,96 @@ All these sentences have the same <em>intent</em>: you need to do the dishes.
-![Emptyquote](../images/alexa_hello_world/emptyquote.png) 
+
+    ![Emptyquote](../images/alexa_hello_world/emptyquote.png){:.enlargeImage}
+    <p style="padding-bottom: 7px"></p> 
 
 5. Within the empty quotation block, type “hello world”. Your block should be <strong>“hello world”</strong>. 
-![Blocksdone](../images/alexa_hello_world/blocksdone.png) 
+
+    ![Blocksdone](../images/alexa_hello_world/blocksdone.png){:.enlargeImage}
+    <p style="padding-bottom: 7px"></p>
 
 ## Sending Your Skill to Amazon
 
 Now, we need to send this <strong>skill</strong> to Amazon.
 
 1. At the rightmost part of the screen, there is the “Testing” box. Within it, there is a button labeled <strong>“Send Updates”</strong>. Click this button.
-![Sendskill](../images/alexa_hello_world/sendskill.png) 
+
+    ![Sendskill](../images/alexa_hello_world/sendskill.png){:.enlargeImage} 
+    <p style="padding-bottom: 7px"></p>
 
 2. The gray part of the “Testing” box should change text when loading. Wait a few minutes until the gray part turns white.
 
 Your current screen should look like the following:
 
-![Sentscreen](../images/alexa_hello_world/sentscreen.png) 
+![Sentscreen](../images/alexa_hello_world/sentscreen.png){:.enlargeImage} 
+<p style="padding-bottom: 7px"></p>
 
 ## Testing your Skill
 
 Now it’s time to test our Alexa skill! If you have an Alexa-enabled device handy and [set up](https://docs.google.com/document/d/15SbjSasmgiCibz_NK8rS0Le1hZ1dZNZSb7a5JIT8H1c/edit), say *“Alexa, open hello world greeter”,* and then *“Alexa, tell hello world greeter to say hello.”* to it and hear the response, which should be “hello world”, as we programmed it.
 
 If you don’t have an Alexa-enabled device on hand, App Inventor allows you to simulate an Alexa using your custom Alexa skill right in your browser! Simply type in the textbox at the bottom, *“Alexa, ask hello world greeter to say hi.”* The response should be what we plugged into the <strong>say</strong> block earlier, *“hello world.”*
 
-![Sayhello](../images/alexa_hello_world/sayhello.png) 
+![Sayhello](../images/alexa_hello_world/sayhello.png){:.enlargeImage} 
+<p style="padding-bottom: 7px"></p>
 
 Note: If you are interested in testing in other ways, feel free to explore this [document](https://docs.google.com/document/u/0/d/1ISS1lI1MK7PCehxCRS3eKQkBiA0fjeXwuL_WlAY-fQU/edit)!
 
 ## Finish!
 
 Congratulations! You’ve made your first custom Alexa Skill! Feel free to extend this app by adding new intents and new ways for Alexa to respond to each intent.
 
+# Expand Your App
+
+Here are some ideas for ways to enhance your app!
+
+* Our Alexa app currently only has one intent- saying "hello world." Try adding more intents that instruct Alexa to say different things. For example, you can add an intent for Alexa to say "good bye."
+
+* Slots: (add something here)
```

Comment:
Have Alexa learn the name of the user from an input user sentence like "my name is Sebastian" and respond "It is my pleasure to meet you Sebastian"  Hint: you will need to learn about Slots.